system-overview/cryptography.md

```diff
@@ -0,0 +1,115 @@
+
+### Types of keys
+
+The two categories of keys are your Azimuth/Ethereum keys and your networking
+keys. In both cases, these are public/private key pairs utilized for public key
+cryptography.
+
+#### Azimuth keys
+
+Your Urbit ID exists as an ERC-721 non-fungible token on the Ethereum
+blockchain, and as such is contained in a wallet whose private key you possess.
+If you are are utilizing a [master
+ticket](@/docs/azimuth/azimuth.md#master-ticket), this private key is derived
+from a seed, which is what you use to login to
+[Bridge](@/docs/glossary/bridge.md). Otherwise, you have generated the key by
+some other process, of which there are too many to list here. Besides the
+private key which unlocks your ownership wallet address, you may have a few
+other private keys which unlock a wallet that corresponds to your ship's
+[proxies](@/docs/glossary/proxies.md). We refer collectively to these keys as
+your _Azimuth keys_.
+
+Only [planets](@/docs/glossary/planet.md), [stars](@/docs/glossary/star.md), and
+[galaxies](@/docs/glossary/galaxy.md) have Azimuth keys.
+[Moons](@/docs/glossary/moon.md) and [comets](@/docs/glossary/comet.md) do not,
+as they do not utilize the Ethereum blockchain.
+
+It is important to note that no Azimuth keys are stored anywhere within your
+ship's [pier](@/docs/glossary/pier.md) - Ethereum and Urbit ID are entirely
+separate entities from Urbit itself, and so you lose access to your Azimuth
+private keys there is no way to retrieve them somehow from your ship.
+
+For more information on the usage of these keys and the associated proxies, see
+the [Azimuth documentation](@/docs/azimuth/azimuth.md).
+
+#### Networking keys
+
+All communications in Urbit over the [Ames](@/docs/glossary/ames.md) networking
+are end-to-end encrypted, and thus your ship stores its own public/private pair
+of _networking keys_ utilized for encryption and authentication. Networking keys
+for all ship types are stored within the ship's Jael vane.
+
+For planets, stars, and galaxies, your private networking key is a
+necessary input (the [keyfile](@/docs/glossary/keyfile.md)) for the initial boot
+sequence of your ship (sometime called its `%dawn`). The associated public key
+is then uploaded to the Ethereum blockchain using one of your Azimuth private
+keys - either the one associated to the wallet which owns the ship, or the one
+which holds the management proxy.
+
+For moons, the private networking key is generated by the parent ship and
+injected into the `%dawn` sequence for the moon, and its public key is stored by
+the parent in its Jael vane. Moon networking keys may be changed by the parent
+ship, but not by the moon itself.
+
+For comets, their 128-bit `@p` name is actually their public key, and the
+"mining" process to generate a comet consists of guessing a private key for
+which the associated public key has a galaxy that has already been pre-assigned
+to assist its peer discovery process. Thus, comets cannot change their
```

Comment:
Comet sponsorship is done by stars, actually! The mining then is finding a comet whose sponsor (through the last two bytes of its `@p`) resolves to a star that is on the "comet sponsors list" as downloaded during boot.